setup.py

```diff
@@ -64,9 +64,17 @@
     extras_require['all'].extend(reqs)
 
 install_requires = [
-    'Invenio[base,auth,metadata,files]{}'.format(invenio_version),
-    'invenio-rdm-records>=0.9.0',
+    'Invenio[base,auth,files]{}'.format(invenio_version),
+    'invenio-indexer>=1.1.1,<1.2.0',
+    'invenio-jsonschemas>=1.0.1,<1.1.0',
+    'invenio-oaiserver>=1.1.1,<1.2.0',
+    'invenio-pidstore>=1.1.0,<1.2.0',
+    'invenio-records-rest>=1.6.4,<1.7.0',
+    'invenio-records-ui>=1.0.1,<1.1.0',
+    'invenio-records>=1.3.0,<1.4.0',
+    'invenio-rdm-records>=0.0.9',
     'invenio-records-permissions>=0.7.0',
+    'werkzeug>=0.16.1,<1.0.0',
```

Comment:
Is this needed? I think is fixed already in other packages if not just bump babelex and invenio-accounts.

Reply:
I didn't know if it is fixed, is it?